packages/devtools/lib/src/tables.dart

```diff
@@ -533,12 +666,18 @@ abstract class Column<T> {
 
   bool get usesHtml => false;
 
-  /// Get the cell's value from the given [item].
-  dynamic getValue(T item);
+  /// Get the cell's value from the given [dataObject].
+  dynamic getValue(T dataObject);
```

Comment:
make this type Comparable and implicitly we require the value to be Comparable for sorting.